src/plugins/intel_gna/gna_plugin.cpp

```diff
@@ -748,8 +752,9 @@ void GNAPlugin::LoadNetwork(CNNNetwork & _network) {
         pass_config->disable<ngraph::pass::TransposeReduction>();
         manager.run_passes(graph);
         convertedNetwork = InferenceEngine::details::convertFunctionToICNNNetwork(graph, clonedNetwork);
-
         isNgraphPassesUsed = true;
+        // saving modified graph (it should be deprecated when migration to ngraph finished)
+        model = graph;
```

Comment:
I think it is better to keep only necessary information instead of duplication of the whole graph.